lib/components/job_widget.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+class JobWidget extends StatelessWidget {
+  final String requesterName;
+  final String description;
+  final int phoneNumber;
+  final GeoPoint location;
```

Comment:
Is this GeoPoint extracted from the phone location? Or is this from the existing library that EditLocation uses?

Reply:
It's from one of the Firebase objects.